lib/tasks/gettext.rake

```diff
@@ -17,9 +17,19 @@ namespace :gettext do
   end
 
   def files_to_translate
-    GettextSetup.config['source_files'].map do |p|
+    files = GettextSetup.config['source_files'].map do |p|
       Dir.glob(p)
     end.flatten
+    # check for optional list of files to exclude from string
+    # extraction
+    files_to_exclude = GettextSetup.config['exclude_files']
+    if (files_to_exclude)
```

Comment:
Ah - I hadn't realized that bit was controversial :) ah well. so many ways to do the same thing in ruby.